app/javascript/api/apiEndpoints.ts

```diff
@@ -0,0 +1,5 @@
+export const listing = (listingId) => `/api/v1/listings/${listingId}.json`
+export const listingPreferences = (listingId) => `/api/v1/listings/${listingId}/preferences`
+export const lotteryBuckets = (listingId) => `/api/v1/listings/${listingId}/lottery_buckets`
+export const lotteryRanking = (listingId, lotteryId) =>
+  `/api/v1/listings/${listingId}/lottery_ranking?lottery_number=${lotteryId}`
```

Comment:
i was hoping to use [msw](https://mswjs.io/) to mock up network calls, but looks like we need to update some libs first